hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/file/TestOMFileCreateRequest.java

```diff
@@ -115,57 +118,69 @@ public void testPreExecuteWithBlankKey() throws Exception {
 
   @Test
   public void testValidateAndUpdateCache() throws Exception {
-    OMRequest omRequest = createFileRequest(volumeName, bucketName, keyName,
-        HddsProtos.ReplicationFactor.ONE, HddsProtos.ReplicationType.RATIS,
-        false, true);
 
-    OMRequestTestUtils.addVolumeAndBucketToDB(volumeName, bucketName,
-        omMetadataManager, getBucketLayout());
-    OMFileCreateRequest omFileCreateRequest = getOMFileCreateRequest(omRequest);
+    String[] validKeyNames = {
+        keyName,
+        OM_SNAPSHOT_INDICATOR + "abc/" + keyName,
+        "a/" + OM_SNAPSHOT_INDICATOR + "/b/c/" + keyName
+    };
 
-    OMRequest modifiedOmRequest = omFileCreateRequest.preExecute(ozoneManager);
+    for (String validKeyName : validKeyNames) {
+      keyName = validKeyName;
```

Comment:
The only changes I added in this tests is just to create a String array of validKeyNames, then iterate them through as keyName in original test. (also add indent to original code block due to the new for-loop)